biz.aQute.bndlib/src/aQute/bnd/osgi/repository/SimpleIndexer.java

```diff
@@ -202,7 +202,11 @@ private Resource indexFile(File file) {
 
 	private URI relativize(File file) {
 		if (base == null) {
-			return file.toURI();
```

Comment:
I am not sure this is a good idea. The caller supplies the File objects and can supply canonical/absolute/relative File objects if they choose.